docs/versioned-plugins/codecs-index.asciidoc

```diff
@@ -35,8 +35,6 @@ include::codecs/json-index.asciidoc[]
 
 include::codecs/json_lines-index.asciidoc[]
 
-//include::codecs/json_pretty-index.asciidoc[]
```

Comment:
I've commented out the include statements for index files that are missing from the PR.